spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -1422,6 +1422,14 @@ def create_iap!(app_id: nil, type: nil, versions: nil, reference_name: nil, prod
       handle_itc_response(r.body)
     end
 
+    # Creates or retrieves app-specific shared secret key
+    # :get - retrieve, :post - create
+    def handle_shared_secret(method: nil, app_id: nil)
+      r = request(method, "ra/apps/#{app_id}/iaps/appSharedSecret")
+      data = parse_response(r, 'data')
+      data['sharedSecret']
+    end
```

Comment:
Can we actually split this into two methods? One for the `:get` and one for the `:post`. I’d just like this to follow the rest of the conventions in this class and keep the `:get` and `:post` encapsulated into this file 😇

Reply:
Thanks for review, totally agree. Have done this! 👍